### DISCUSSION TITLE
preventing libreoffice from taking whole screen

### DISCUSSION THREAD

**Discussion by pescepalla** [hyprwm/Hyprland#5264] [Q&A] [answered]
Libreoffice has the bad habit of covering other windows when opening. Earlier, I managed this behaviour with

`windowrulev2 = nomaximizerequest, class:^libreoffice.*$`

However, this has stopped working since 0.37 and it looks like `nomaximizerequest` was taken off the [Window Rules](https://wiki.hyprland.org/Configuring/Window-Rules/).

How do I prevent `libreoffice` from covering all other windows now?

**Comment by MightyPlaza** [comment, marked as answer]
https://github.com/hyprwm/hyprland-wiki/pull/545

**Comment by opalmay** [comment]
Updated variation:
`windowrule = match:class ^(libreoffice.*)$, suppress_event maximize`
